README.md

```diff
@@ -164,21 +164,17 @@ int fib(int n)      fib:                        Reserve stack frame for function
 
 ## Known Issues
 
-1. The compiler uses different logic to evaluate the operand of `if` or `!` than a general integer
-   expression. Therefore, explicit failback to "evaluate value and compare against zero" is required
-   if the operand is not a "truthiness-generating" expression.
-   That is, the expression `0 == strcmp(ptr, "hello")` is not equivalent to `!strcmp(ptr, "hello")`.
-2. The generated ELF lacks of .bss and .rodata section
-3. The unary `*` operator is not supported, which makes it necessary to use `[0]` syntax.
+1. The generated ELF lacks of .bss and .rodata section
+2. The unary `*` operator is not supported, which makes it necessary to use `[0]` syntax.
    Consider `int x = 5; int *ptr = &x;` and it is forbidden to use `*ptr`. However, it is valid to
    use `ptr[0]`, which behaves the same of `*ptr`.
-4. The support of varying number of function arguments is incomplete. No `<stdarg.h>` can be used.
```

Comment:
You don't have to change the numbers. Markdown parser can still retrieve from non-continuous numbers. You can simply remove the first item.